app/src/main/java/com/nmakademija/nmaakademija/OnboardingActivity.java

```diff
@@ -0,0 +1,49 @@
+package com.nmakademija.nmaakademija;
+
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.v7.preference.PreferenceManager;
+import android.widget.Toast;
+
+import com.nmakademija.nmaakademija.fragment.OnboardingFragment;
+
+public class OnboardingActivity extends BaseActivity {
+
+    private boolean doubleBackToExitPressedOnce = false;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_onboarding);
+
+        getSupportActionBar().hide();
+
+        getSupportFragmentManager().beginTransaction()
+                .replace(R.id.main_frame, new OnboardingFragment())
+                .commit();
+    }
+
+    @Override
+    public void onBackPressed() {
+        if (!PreferenceManager.getDefaultSharedPreferences(getApplicationContext())
+                .getString(getString(R.string.section_key), "0")
+                .equals("0")) {
+            finish();
+        } else if (doubleBackToExitPressedOnce) {
```

Comment:
I don't see any reason to have double click handling logic. What was you intentions about it ?